src/components/molecules/Splash.tsx

```diff
@@ -0,0 +1,38 @@
+import LogoFesbalWhiteIcon from "../icons/LogoFesbalWhiteIcon";
+import {useTranslation} from "react-i18next";
+import {namespaces} from "../../i18n/i18n.constants";
+import SplashCard from "../atom/SplashCard";
+
+const Splash = () => {
+    const {t: translate} = useTranslation(namespaces.pages.welcomeScreen);
+    const FIGMA_PROTOTYPE_ENTITIES_LOGIN_URL = "https://www.figma.com/proto/fieW8OoWlFLyL90UleIeQ1/Draft-0.6---FESBAL?page-id=1147%3A14538&node-id=1147%3A15887&viewport=4483%2C1878%2C0.53&scaling=min-zoom&starting-point-node-id=1147%3A15887"
```

Comment:
I think this might be kinda weird for the user. Are we exposing the figma to the general public???